packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -670,6 +671,7 @@ export async function pollTransformationJob(jobId: string, validStates: string[]
                 })
             }
             transformByQState.setPolledJobStatus(status)
+            getLogger().info(`CodeTransformation: polled job status = ${status}`)
```

Comment:
FYI: `getLogger()` now has a "topic" feature so you can do this at the top of the module:

    const logger = getLogger('CodeTransformation')

then `logger.info()` will prepend the topic name.